license.txt

```diff
@@ -0,0 +1 @@
+eyJhbGciOiJFZERTQSIsImtpZCI6IjIwMjItMDgtMTIiLCJ0eXAiOiJKV1QifQ.eyJpc3MiOiJtYWNoaW5lQGNvZGVyLmNvbSIsImV4cCI6MTY3MTQ3OTgyOCwibmJmIjoxNjY1MDg2MjI1LCJpYXQiOjE2NjUwODYyMzQsImxpY2Vuc2VfZXhwaXJlcyI6MTY3MDI3MDIyOCwiYWNjb3VudF90eXBlIjoic2FsZXNmb3JjZSIsImFjY291bnRfaWQiOiIwMDExVTAwMDAxTWVQaW9RQUYiLCJ0cmlhbCI6ZmFsc2UsInZlcnNpb24iOjMsImZlYXR1cmVzIjp7InVzZXJfbGltaXQiOjAsImF1ZGl0X2xvZyI6MSwiYnJvd3Nlcl9vbmx5IjoxLCJzY2ltIjoxLCJ3b3Jrc3BhY2VfcXVvdGEiOjEsInJiYWMiOjF9fQ.wCVLOmifUVXvyq8Wos7iS6vN0GilIaaAOQhEMnsrkyGCE57jNC46KH1kHdJf8l6ZduxgJC7blHqRCkigJz-GDw
```

Comment:
I imagine this shouldn't be here

Reply:
My bad! Can we invalidate that?

Reply:
I will force a push to remove that from git